components/transport_profiles/app/models/transport_profiles/steps/route_to.rb

```diff
@@ -22,6 +22,14 @@ def initialize(endpoint_key, file_name, gateway, destination_file_name: nil, sou
       @source_credentials = resolve_source_credentials(source_credentials)
```

Comment:
@TreyE to make it less confusing, I think it makes it  more sense to call the argument file_name actually be file_uri